Makefile

```diff
@@ -128,6 +129,7 @@ clean:
 
 binaries: $(BUILD_DIR)/containerd $(BUILD_DIR)/ctr
 
+static-binaries: GOARCH=$(ARCH)
```

Comment:
This will not work somebody run `make static-binaries` directly as we are not assigning anything to ARCH by default.

Reply:
Just check whether we can directly pass GOARCH instead of ARCH while running make targets.

Reply:
Just check whether we can use GOARCH itself instead of ARCH while calling make targets.